src/widgets/DropdownRow.js

```diff
@@ -66,7 +63,13 @@ export const DropdownRow = ({
         />
       </View>
       <View style={localStyles.flexNine}>
-        <SimpleLabel size={labelSize} text={currentOptionText} />
+        <TextInput
+          onChangeText={onSelection}
+          value={currentOptionText}
+          underlineColorAndroid={SUSSOL_ORANGE}
+          style={{ flex: 1, fontFamily: APP_FONT_FAMILY }}
+          placeholder="Usage direction"
```

Comment:
this is very specific within a general component. A temporary thing?
I was expecting props to enable the text input and supply the placeholder

Reply:
whoops! thanks : 1ac1107690ce2800fccf792bf8eb9e23906ea09c